services/core-web/src/components/mine/Permit/MinePermitTable.tsx

```diff
@@ -188,494 +192,575 @@ const renderPermitNo = (permit) => {
-    lastAmendedVC: latestAmendment?.vc_credential_exch_state,
-    permitNo: permit.permit_no || Strings.EMPTY_FIELD,
-    firstIssued: (firstAmendment && formatDate(firstAmendment.issue_date)) || Strings.EMPTY_FIELD,
-    permittee: permit.current_permittee,
-    permit_amendments: permit.permit_amendments.filter(
-      (a) => a.permit_amendment_status_code !== draftAmendment
-    ),
-    status: permit.permit_status_code,
-    addEditButton: {
-      major_mine_ind,
-      hasAmalgamated,
+            <Dropdown overlay={menu} placement="bottomLeft">
+              {/* @ts-ignore */}
+              <Button type="secondary" className="permit-table-button">
+                Actions
+                <img
+                  className="padding-sm--right icon-svg-filter"
+                  src={CARAT}
+                  alt="Menu"
+                  style={{ paddingLeft: "5px" }}
+                />
+              </Button>
+            </Dropdown>
+          </div>
+        );
+      },
     },
-    openEditPermitModal,
-    openAddPermitAmendmentModal,
-    openAddAmalgamatedPermitModal,
-    permitStatusOptionsHash,
-    permit,
-    handleDeletePermit,
-    handleDeletePermitAmendment,
-    openAddPermitHistoricalAmendmentModal,
-    openEditSitePropertiesModal,
-  };
-};
+  ];
 
-const transformChildRowData = (
-  amendment,
-  record,
-  amendmentNumber,
-  major_mine_ind,
-  openEditAmendmentModal,
-  handleDeletePermitAmendment,
-  handlePermitAmendmentIssueVC,
-  permitAmendmentTypeOptionsHash,
-  openViewConditionModal,
-  mineGuid
-) => ({
-  amendmentNumber,
-  isAmalgamated: amendment.permit_amendment_type_code === PERMIT_AMENDMENT_TYPES.amalgamated,
-  receivedDate: formatDate(amendment.received_date) || Strings.EMPTY_FIELD,
-  issueDate: formatDate(amendment.issue_date) || Strings.EMPTY_FIELD,
-  authorizationEndDate: formatDate(amendment.authorization_end_date) || Strings.EMPTY_FIELD,
-  description: amendment.description || Strings.EMPTY_FIELD,
-  isAssociatedWithNOWApplicationImportedToCore: "",
-  openEditAmendmentModal,
-  openViewConditionModal,
-  permit: record.permit,
-  documents: amendment.related_documents,
-  handleDeletePermitAmendment,
-  handlePermitAmendmentIssueVC,
-  finalApplicationPackage: finalApplicationPackage(amendment),
-  maps: amendment.now_application_documents?.filter(
-    (doc) => doc.now_application_document_sub_type_code === "MDO"
-  ),
-  permitAmendmentTypeOptionsHash,
-  permitAmendmentDocuments: amendment.related_documents.map((doc) => ({
-    key: doc.mine_report_submission_guid,
-    documentManagerGuid: doc.document_manager_guid,
-    filename: doc.document_name,
-  })),
-  mineGuid,
-  ...amendment,
-});
-
-export const MinePermitTable: React.FC<RouteComponentProps & MinePermitTableProps> = (props) => {
   const permitColumns = [...columns];
 
+  const partyRelationships = useSelector(getPartyRelationships);
```

Comment:
This is what I think happened:
1. MinePermitTable was made, and used redux mapStateToProps to get partyRelationships from the state as a "prop". It was then added to propTypes
2. Someone working on MinePermitInfo saw partyRelationships in the propTypes and passed it as a prop
3. Leading to it being both a prop and a redux state value

We could do either/or, really: take it out as a prop, or take out the useSelector redux value. I'd probably take it out as a prop because MinePermitInfo isn't otherwise using `partyRelationships`